cfy_manager/components/usage_collector/scripts/script_utils.py

```diff
@@ -0,0 +1,85 @@
+import json
+import logging
+import pkg_resources
+
+from os import path
+from uuid import uuid4
+from requests import post
+from contextlib import contextmanager
+from logging.handlers import WatchedFileHandler
+
+from manager_rest import config, server, premium_enabled
+from manager_rest.storage import get_storage_manager, models
+
+
+MANAGER_ID_PATH = '/etc/cloudify/.id'
+CLOUDIFY_IMAGE_INFO = '/opt/cfy/image.info'
+RESTSERVICE_CONFIG_PATH = '/opt/manager/cloudify-rest.conf'
+LOGFILE = '/var/log/cloudify/usage_collector/usage_collector.log'
+
+logger = logging.getLogger('usage_collector')
+logger.setLevel(logging.INFO)
+formatter = logging.Formatter(fmt='%(asctime)s [%(levelname)s] '
+                                  '[%(name)s] %(message)s',
+                              datefmt='%d/%m/%Y %H:%M:%S')
+file_handler = WatchedFileHandler(filename=LOGFILE)
+file_handler.setFormatter(formatter)
+logger.addHandler(file_handler)
+
+
+@contextmanager
+def get_storage_manager_():
```

Comment:
Why does it end with `_`?

Reply:
Originally it began with _ but this function is no longer private, and still needs to be distinguished from
get_storage_manager we import from manager_rest